plugins/woocommerce-blocks/assets/js/blocks/add-to-cart-with-options/index.tsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import { registerBlockSingleProductTemplate } from '@woocommerce/atomic-utils';
+import { Icon, button } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import metadata from './block.json';
+import AddToCartFormEdit from './edit';
```

Comment:
should we use the `AddToCartOptionsEdit` name?

```suggestion
import AddToCartOptionsEdit from './edit';
```

Reply:
@retrofox Done! I also renamed `AddToCartFormSettings` to `AddToCartOptionsSettings`.